tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php

```diff
@@ -97,11 +97,10 @@ private function test123(){
 EOF;
         $this->doTest($this->removeLinesFromString($template, $lineNumberRemoved), $template);
 
-        if (null === $config) {
-            $this->fixer->configure(array());
-        } else {
-            $this->fixer->configure(array('tokens' => $config));
-        }
+        null === $config
```

Comment:
happy with revert, all good, out of curiosity what is this leaking side effect (just trying to understand)